src/analysis/souffle/tests/arcs_manifest_tests_todo/BUILD

```
@@ -0,0 +1,15 @@
+load("//build_defs:arcs.bzl", "arcs_manifest_proto")
+
+# TODO(Issue #61): Enable translating these manifests to proto once manifest2proto can parse them.
+BUILD_ERROR_ARCS_MANIFEST_PATTERNS = [
+    "*fail_field_nullable_inline_entity_direct.arcs",
+    "*ok_join_simple_with_nullables.arcs",
+]
+
+BUILDING_ARCS_MANIFESTS = glob(["*.arcs"], BUILD_ERROR_ARCS_MANIFEST_PATTERNS)
```

Let us be explicit about the second parameter: 
```suggestion
BUILDING_ARCS_MANIFESTS = glob(
   ["*.arcs"], 
   excludes = BUILD_ERROR_ARCS_MANIFEST_PATTERNS
)
```

---

Done.